registry/repositories.go

```diff
@@ -232,16 +278,28 @@ func streamHarborProjectsPage(ctx context.Context, registry *Registry, c chan st
 			continue
 		}
 
+		var harborProjRepo string = ctx.Value("harborProjRepo").(string)
+
+		if strings.Contains(harborProjRepo, "v2.0") {
```

Comment:
Is this safe? Is it possible for `harborProjRepo` to include a `v2.0` that's not a result of `/api/v2.0/projects`?

Reply:
Changed to use regex now. Thanks for the feedback!